tool.go

```diff
@@ -750,7 +750,17 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 	}
 
 	if runtime.GOOS != "windows" {
-		allArgs = append(allArgs, "--stack_size=10000", script)
+		r := syscall.Rlimit{}
+
+		err := syscall.Getrlimit(syscall.RLIMIT_STACK, &r)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "gopherjs: failed to get stack size limit: %v", err)
+			os.Exit(1)
```

Comment:
Returning an error in this case would appear to enter the flow of `handleError`, `printError` (which also handles the newline where required) - I'm not so familiar with how you look to handle this sort of "system" error vs. compile errors, etc. Any thoughts on what you'd prefer?